metadata/transportation/railroads.md

```diff
@@ -0,0 +1,72 @@
+# Title
+
+Utah Railroads
+
+## ID
+
+0b2a9859-5b4e-454b-a7ab-94e54ca13671
+
+## Brief Summary
+
+Polyline dataset of active and abandoned railway lines in Utah.
+
+## Summary
+
+This dataset contains polylines representing railway lines in Utah. Features in this dataset includes commercial railroads, commuter lines, and other forms of rail transportation. Information such as current railroad availability, pricing, and schedule, are not included with this layer.
+
+## Description
+
+### What is the dataset?
+
+This dataset contains polyline features representing active and abandoned railway lines in Utah.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general analytic and cartographic purposes.
+
+### What does the dataset represent?
+
+Each feature in this dataset includes attributes such as railway status, track type (light, heavy, or electrified), and current operator.
+
+### How was the dataset created?
+
+UGRC created this dataset using high resolution imagery from a variety of sources including the National Agriculture Imagery Program ([NAIP](https://naip-usdaonline.hub.arcgis.com/)) and [USGS orthoimagery](https://www.usgs.gov/faqs/what-digital-orthophoto-quadrangle-doq-or-orthoimage).
```

Comment:
Should we link to our NAIP here? 

https://gis.utah.gov/products/sgid/aerial-photography/naip/

Reply:
I think so. I'll add that in.